bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/Arbitraries.scala

```diff
@@ -43,30 +45,39 @@ trait Arbitraries extends edu.gemini.spModel.core.Arbitraries {
   implicit def arbNonEmptyList[A: Arbitrary]: Arbitrary[NonEmptyList[A]] =
     Arbitrary {
       for {
-        a  <- arbitrary[A]
+        a <- arbitrary[A]
         as <- arbitrary[List[A]]
       } yield NonEmptyList.nel(a, as)
     }
 
+  implicit def arbImList[A: Arbitrary]: Arbitrary[ImList[A]] =
+    Arbitrary {
+      arbitrary[List[A]].map(_.asImList)
+    }
+
   implicit def arbOptsList[A: Arbitrary]: Arbitrary[OptsList[A]] =
     Arbitrary {
       arbitrary[NonEmptyList[A] \/ Zipper[A]].map { d => OptsList(d) }
     }
 
   implicit val arbGuideProbe: Arbitrary[GuideProbe] =
-    Arbitrary { oneOf(GuideProbeMap.instance.values().asScala.toList) }
+    Arbitrary {
+      oneOf(GuideProbeMap.instance.values().asScala.toList)
+    }
 
   implicit val arbManualGroup: Arbitrary[ManualGroup] =
     Arbitrary {
       for {
         n <- alphaStr
-        m <- listOf(arbitrary[(GuideProbe, OptsList[SPTarget])]).map { _.toMap }
+        m <- listOf(arbitrary[(GuideProbe, OptsList[SPTarget])]).map {
+          _.toMap
+        }
```

Comment:
Trivial, but I find this formatting harder to read.  Here it might just be `.map(_.toMap)` but in general if it goes over one line I'd rather see the closing brace line up with the `listOf` above it than spill into the column occupied by the `val`s.  I don't know though, maybe that's weird.